blocks/library/block/index.js

```diff
@@ -172,20 +172,10 @@ export const name = 'core/block';
 
 export const settings = {
 	title: __( 'Shared Block' ),
```

Comment:
What about the title and description? It seems to me that blocks discovery would benefit from these properties being declared on the server as well

Reply:
I'm going back and forth with those two. After another round of musings, I think it makes sense to move them to the server. In addition, we should move non-SVG version of the icon. This might allow lazy loading of blocks on the client because we would have something to display in the inserters.

Reply:
I moved `title`, `description`, `icon` and `keywords` to the server, too.